### DISCUSSION TITLE
Solution's flexibility

### DISCUSSION THREAD

**Discussion by dilivar** [HansvBug/Bitvavo-Coin-Monitor#2] [Ideas]
Hi Hans,
  I wonder if your monitor tool can work for any other exchange if I only change the API url for ticker?

**Comment by HansvBug** [comment]
Hi

I did not test it I lack the time to continue working on it.



Op donderdag 13 januari 2022 schreef dilivar ***@***.***>:

**Comment by dilivar** [comment]
Hi Hans;
  I wonder if you can help me about Bitvavo. I can not create local order book with Bitvavo web socket. I start with getbook request via web socket then I listen orderbook diff and if I miss orderbook I re-request getbook but this time it's empty. 
  Is there any chance that you can share with me a working C# example for creating local orderbook of Bitvavo?

**Comment by HansvBug** [comment]
Hi,

I haven't been looking at the order book. Can you send me the code you all
ready have for creating a local order book. Then i don't need to start
fresh.
Are you from the Netherlands?

Greatings

Op zo 24 jul. 2022 om 23:48 schreef dilivar ***@***.***>:
